### PR TITLE
Annotation refactor annotaion information on click

### PR DESCRIPTION
**Pull request by Premmie**

### CODE REVIEW

File: src/main/java/gui/GraphDrawer.java [hunk 1]
```diff
@@ -255,75 +255,90 @@ private void drawNodes() {
...
+            return anno;
+        }
+        int startBucket = (node.getOffsets()[annotatedGenomeIndex] / BUCKET_SIZE);
+        int endBucket = ((node.getOffsets()[annotatedGenomeIndex] + node.getSequenceLength()) / BUCKET_SIZE);
+        for (int i = startBucket; i <= endBucket; i++) {
+            HashSet<Annotation> tempAnnotations = allAnnotations.get(i);
+            if (tempAnnotations != null) {
+                anno.addAll(tempAnnotations);
             }
-            int[] bigOne = new int[2];
```
Comment (LBoleij) [inline src/main/java/gui/GraphDrawer.java]: Jip, Pls

**Comment by LBoleij** [comment]
Maybe instead of clicking on nodes, click on the specific annotation? And give the information somewhere else?

**Comment by JaspervanTilburg** [comment]
Information about annotations is unclear

**Comment by Jipje** [comment]
Okay added the clicking and some merges were harder than expected. But we're here.

**Comment by LBoleij** [comment]
All works nicely :)

**Pull request merged**